MonoGame.Framework/WindowsUniversal/UAPGamePlatform.cs

```diff
@@ -132,13 +132,22 @@ public override void RunLoop()
 
         public override void StartRunLoop()
         {
-            CompositionTarget.Rendering += (o, a) =>
+            CoreWindow.GetForCurrentThread().Dispatcher.RunIdleAsync((a) =>
             {
-				UAPGameWindow.Instance.Tick();
-                GamePad.Back = false;
-            };
+                CoreWindow.GetForCurrentThread().Dispatcher.RunAsync( CoreDispatcherPriority.Low, OnRenderFrame);
+            });
```

Comment:
This looks sort of weird... why are you dispatching once with `RunIdleAsync` to just dispatch again `RunAsync(Low)`?

Seems easier to just do:
```
public override void StartRunLoop() 
{
   OnRenderFrame();
}
```
This will do one tick to kick things off then register itself for the next.